conf/tax_years.conf

```diff
@@ -107,9 +107,9 @@ tax-years = [
     income-tax-higher-rate-threshold = 37500
     lettings-relief-max-threshold = 40000
     cgt-rates {
-      lower-band-residential      = 18
```

Comment:
I try to follow equal spacing (just like before changes) cause I believe it's clearer to read. Is any of the approaches the industry standard or is it just a matter of preferences? I'm just asking to know for my future PRs

Reply:
I just used the auto formatter in Intellij